pkg/systemd/quadlet/quadlet.go

```diff
@@ -567,7 +569,7 @@ func ConvertContainer(container *parser.UnitFile, isUser bool) (*parser.UnitFile
 	update, ok := container.Lookup(ContainerGroup, KeyAutoUpdate)
 	if ok && len(update) > 0 {
 		podman.addLabels(map[string]string{
-			"io.containers.autoupdate": update,
+			define.AutoUpdateLabel: update,
```

Comment:
importing define seems to bloat this a lot, I agree that we should not duplicate strings so you have to move the definition in another package

Reply:
I think we should examine why define is bloating.  Quadlet size does not make much difference.

Reply:
The size will matter very much, literally the first sentence in in the systemd man page:
> Generators are small executables ...

The generator blocks early boot, the more bloat we have the slower the binary will run and thus result in slower boot times.
Based on a quick benchmark which just uses -dryrun it doubles the execution time.

main:
```
$ hyperfine --warmup 10  -N 'bin/quadlet-main -dryrun'
Benchmark 1: bin/quadlet-main -dryrun
  Time (mean ± σ):       1.4 ms ±   0.3 ms    [User: 0.4 ms, System: 1.2 ms]
  Range (min … max):     1.1 ms …   4.3 ms    1969 runs
```
this PR:
```
$ hyperfine --warmup 10  -N 'bin/quadlet -dryrun'
Benchmark 1: bin/quadlet -dryrun
  Time (mean ± σ):       3.0 ms ±   0.5 ms    [User: 1.2 ms, System: 2.0 ms]
  Range (min … max):     2.5 ms …   7.2 ms    885 runs
```

So yeah sure 1.5 ms is not much but the whole point of the bloat check it to catch these things and not ignore them.

Reply:
I agree but we need to figure how how libpod/define is causing bloat.  The define section is supposed to be used to prevent bloat.  I can fix it hear and sweep it under the rug, but until someone looks at define to see where it got bloat...